src/commcare_cloud/ansible/roles/couchdb2/templates/extras.ini.j2

```diff
@@ -14,3 +14,10 @@ use_kill_all = {{ couchdb_rexi_use_kill_all }}
 os_process_timeout = 20000
 ; set limit to previous default of 4GB on 2.3.1
 max_document_size = 4000000000
+
+; compaction settings
+[smoosh.ratio_dbs]
+min_priority = 1.3
+
+[smoosh.ratio_views]
+min_priority = 1.3
```

Comment:
Just to make sure I'm reading this right, conceptually this means views and dbs are compacted once they become 30% than they "need to be", right?

Reply:
Yeah, when the size of the db or view is 130% of the size of active data in that db (30% is deleted/updated/etc).